src/redux/store/initialState.js

```diff
@@ -10,10 +10,12 @@ export default {
     updateProfileSuccess: {},
     updateProfileError: null,
   },
+  viewArticle: {
```

Comment:
I suggest naming it generous names like an article as we all going use it by deleting it updating and reporting.

Reply:
could you please suggest a generous name?

Reply:
like `article`

Reply:
@nkalyesubula  is this resolved?